go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/rhobs/kube-events-exporter
 
 go 1.14
+
+require github.com/golangci/golangci-lint v1.24.0
```

Comment:
Since this is golang 1.14, you should use a separate `go.mod` file for tooling. More about it in https://golang.org/doc/go1.14#tools

Reply:
I moved every file related to tooling modules inside of the `scripts` directory